packages/headless/src/features/product-listing/product-listing-actions.ts

```diff
@@ -125,12 +132,54 @@ export const buildProductListingRequest = (
           ) + 1,
       },
     }),
-    ...(state.sortCriteria && {
-      sortCriteria: state.sortCriteria,
+    ...(state.sort && {
```

Comment:
Wondering if this should check for `state.sort && state.sort.by !== "relevance"` since it is the default value

Reply:
Could be a nice optimization indeed.